demos/icon-button.html

```diff
@@ -0,0 +1,280 @@
+                  </svg>
+                </button>
+              </div>
+            </div>
+
+            <div class="toggle-example">
+              <h2 class="mdc-typography--headline6">Disabled Buttons</h2>
+              <div class="demo-wrapper">
+                <button class="mdc-icon-button material-icons"
+                        aria-label="Airplane Mode active"
+                        aria-pressed="false"
+                        disabled>
+                  airplanemode_active
+                </button>
+                <button class="mdc-icon-button"
+                        aria-label="Add to favorites"
+                        aria-pressed="false"
+                        disabled>
+                  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
+                    <path fill="none" d="M0 0h24v24H0z"/>
+                    <path d="M23 12c0-6.07-4.93-11-11-11S1 5.93 1 12s4.93 11 11 11 11-4.93 11-11zM5 17.64C3.75 16.1 3 14.14 3 12c0-2.13.76-4.08 2-5.63v11.27zM17.64 5H6.36C7.9 3.75 9.86 3 12 3s4.1.75 5.64 2zM12 14.53L8.24 7h7.53L12 14.53zM17 9v8h-4l4-8zm-6 8H7V9l4 8zm6.64 2c-1.55 1.25-3.51 2-5.64 2s-4.1-.75-5.64-2h11.28zM21 12c0 2.14-.75 4.1-2 5.64V6.37c1.24 1.55 2 3.5 2 5.63z"/>
+                  </svg>
+                </button>
+              </div>
+            </div>
+
+            <div class="toggle-example">
+              <h2 class="mdc-typography--headline6">Larger Buttons</h2>
+              <div class="demo-wrapper">
+                <button class="mdc-icon-button material-icons demo-icon-button-large"
+                        aria-label="Airplane Mode active"
+                        aria-pressed="false">
+                  airplanemode_active
+                </button>
+                <button class="mdc-icon-button demo-icon-button-large"
+                        aria-label="Add to favorites"
+                        aria-pressed="false">
+                  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
+                    <path fill="none" d="M0 0h24v24H0z"/>
+                    <path d="M23 12c0-6.07-4.93-11-11-11S1 5.93 1 12s4.93 11 11 11 11-4.93 11-11zM5 17.64C3.75 16.1 3 14.14 3 12c0-2.13.76-4.08 2-5.63v11.27zM17.64 5H6.36C7.9 3.75 9.86 3 12 3s4.1.75 5.64 2zM12 14.53L8.24 7h7.53L12 14.53zM17 9v8h-4l4-8zm-6 8H7V9l4 8zm6.64 2c-1.55 1.25-3.51 2-5.64 2s-4.1-.75-5.64-2h11.28zM21 12c0 2.14-.75 4.1-2 5.64V6.37c1.24 1.55 2 3.5 2 5.63z"/>
+                  </svg>
+                </button>
+              </div>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <section class="example">
+        <div>
+          <div>
+            <h1 class="mdc-typography--headline5">Button Toggles</h1>
+          </div>
+          <div class="toggle-examples-container">
+            <div class="toggle-example">
+              <h2 class="mdc-typography--headline6">Using Material Icons</h2>
+              <div class="demo-wrapper">
+                <a href="#"
```

Comment:
I'd suggest changing this to `button` like the rest in this section. It doesn't make sense for a link to be a toggle, because links navigate.